cg/meta/demultiplex/status_db_storage_functions.py

```diff
@@ -55,6 +56,19 @@ def store_flow_cell_data_in_status_db(
     store.session.commit()
 
 
+def check_if_samples_have_files(flow_cell_path: Path, sample_ids: List[str]) -> None:
```

Comment:
We can consider not passing the sample ids as a parameter here and only pass the `FlowCellDirectoryData`. That way the validation step is self contained and we can move it to where ever in the flow.

I don't think we ever want to call it with any other `sample_ids` than the ones returned from `get_sample_internal_ids_from_sample_sheet`

Reply:
Good idea, will implement 👍

Reply:
implemented